airbyte-integrations/connectors/source-zendesk-sell/build.gradle

```diff
@@ -0,0 +1,8 @@
+plugins {
+    id 'airbyte-python'
+    id 'airbyte-docker'
+}
+
+airbytePython {
+    moduleDirectory 'source_zendesk_sell'
```

Comment:
Remove this file. It isn't used anymore.

Reply:
Done